packages/elvis-colors/gulpfile.ts

```diff
@@ -29,10 +29,10 @@ const generateElvisColorsCss = async () => {
   fileContent += `:root {\n`;
   Object.entries(rootVariables).forEach(([name, color]) => (fileContent += `\t${name}: ${color};\n`));
   fileContent += `}\n`;
-  fileContent += `.e-theme-light,\n:root {\n`;
+  fileContent += `.e-theme-light, .e-alert--global.e-alert--warn *, .e-theme-dark .e-alert--global:not(.e-alert--warn):not(.e-alert--error) *, \n:root {\n`;
   Object.entries(lightVariables).forEach(([name, color]) => (fileContent += `\t${name}: ${color};\n`));
   fileContent += `}\n`;
-  fileContent += `.e-theme-dark, .e-color-background-3, [class*='e-'][class*="--inverted"] {\n`;
+  fileContent += `.e-theme-dark, .e-color-background-3, .e-theme-light .e-alert--global:not(.e-alert--warn):not(.e-alert--error) *, .e-alert--global.e-alert--error *, [class*='e-'][class*="--inverted"] {\n`;
```

Comment:
What does this do?

Reply:
It turns everything inside to light theme or dark theme based on if it is warning, informative or error.